lib/heex_formatter/formatter.ex

```diff
@@ -85,9 +87,35 @@ defmodule HeexFormatter.Phases.Format do
         end
       end
 
+    mode = if tag in ~w(script tag style code pre), do: String.to_atom(tag), else: :normal
```

Comment:
I would say to use to_existing_atom here, but we probably haven't use them previously
```suggestion
    mode = if tag in ~w(script tag style code pre), do: String.to_existing_atom(tag), else: :normal
```

Reply:
actually we're defining them below, so probably to_existing_atom will work here?

Reply:
Not sure if we need it because we are pattern matching only these tags 🤔